lib/src/components/position_component.dart

```diff
@@ -58,24 +67,22 @@ abstract class PositionComponent extends BaseComponent {
   }
 
   /// Get the position that everything in this component is positioned in relation to
-  Vector2 get absoluteCanvasPosition {
+  Vector2 get absoluteParentPosition {
```

Comment:
I know this isn't your change, but I feel that this doc had an abrupt stop maybe whoever wrote this forgot to finish this sentence.

Reply:
I think we can even add a second line here.

if this component has no parent the absolute parent position is the origin, otherwise it's the parents absolute position

Reply:
Fixed